pkgs/by-name/dw/dwlmsg/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  fetchFromGitea,
+  pkg-config,
+  stdenv,
+  wayland,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dwlmsg";
+  version = "git";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "notchoc";
+    repo = "dwlmsg";
+    rev = "3fc9780f4e";
+    hash = "sha256-LTrWb9IxGvm9yUy47+Om9V5GbLygDcEAbVAExGWgFQw=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    wayland
+  ];
+
+  outputs = [
+    "out"
+  ];
+
+  makeFlags = [
+    "PKG_CONFIG=${stdenv.cc.targetPrefix}pkg-config"
+    "PREFIX=$(out)"
+  ];
+
+  strictDeps = true;
+
+  # required for whitespaces in makeFlags
+  __structuredAttrs = true;
+
```

Comment:
the meta info is missing for this port see: https://ryantm.github.io/nixpkgs/stdenv/meta/ i think that is required